### ISSUE TITLE
Tutorial show say something about creating a database

### ISSUE THREAD

**Issue by autarch** [theory/sqitch#315]
I was expecting that the tutorial would tell me how Sqitch creates the database. After a while, I finally figured that I needed to do that before running `sqitch deploy`. It'd be good for the tutorial to address this. Even a sentence or two along the lines "Sqitch doesn't do this, so you should do it yourself" would be helpful.

**Comment by autarch** [comment]
Alternatively, maybe Sqitch should create the database? This is kind of a pain though, as every database does this very differently.

**Comment by theory** [comment]
Yeah, and in a lot of cases people don't even have the ability to create databases. So I just punted on the whole thing.

**Comment by autarch** [comment]
Makes sense

**Comment by theory** [comment]
Which tutorial were you reading, @autarch? The following tutorials each has a line showing the need to create a database:

* [Postgres](/sqitchers/sqitch/blame/master/lib/sqitchtutorial.pod#L157)
* [MySQL](/sqitchers/sqitch/blame/master/lib/sqitchtutorial-mysql.pod#L181)

The others are more dependent on configuration.
